engine/dlib/src/dlib/intersection.cpp

```diff
@@ -98,4 +98,52 @@ bool TestFrustumSphere(const Frustum& frustum, const dmVMath::Vector4& pos, floa
     return true;
 }
 
+// Returns 'false' if the bounding box is off the frustum. Returning 'true' does not guarantee that the object intersects the frustum or is inside it.
+bool TestFrustumOBB(const Frustum& frustum, const dmVMath::Matrix4& world, dmVMath::Vector3& aabb_min, dmVMath::Vector3& aabb_max)
```

Comment:
As an optimisation, I also want you to support the "skip_near_far" boolean that you see in the other functions.

This allows us to skip planes that one sometimes doesn't care about.